### PR TITLE
[CFM-1292] -Extend OG functionality to add users.

### PR DESCRIPTION
**Pull request by ybaras**
[CFM-1292](https://issuetracker.amplexor.com/jira/browse/CFM-1292)

### CODE REVIEW

**Comment by ordavidil** [comment]
@ybaras 
It doesn't seems like this PR is relating to the SA as it should

**Comment by ordavidil** [comment]
@ybaras 
please add screenshots,
thanks.

**Comment by ybaras** [comment]
cfm-1292

File: project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.module [hunk 1]
```diff
@@ -8,6 +8,11 @@
 include_once 'c4m_user.features.inc';
 
 /**
+ * Define administrator role ID.
+ */
+define('C4M_USER_ROLE_ID_ADMINISTRATOR', 4);
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.module]: @ybaras 
remove this.
Use `user_role_load_by_name()` wherever you need to get the _rid_

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 2]
```diff
@@ -131,6 +131,15 @@ function c4m_user_notifications_notify_user_commented($comment) {
  * Email notification about user invitation to group, sent by admin.
  */
 function c4m_user_notifications_notify_user_invited_by_admin($form, $form_state) {
+  // Apply only to SAs.
+  if (!user_has_role(C4M_USER_ROLE_ID_ADMINISTRATOR)) {
+    return;
+  }
+
+  // Send a welcome email if the option is checked in the form.
+  if ($form_state['values']['notifications'][1]) {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: @ybaras what's the `1` stands for ?

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/c4m_user_notifications.module [hunk 3]
```diff
@@ -270,6 +270,23 @@ function c4m_user_notifications_og_role_revoke($entity_type, $gid, $uid, $rid) {
...
+  if (!user_has_role(C4M_USER_ROLE_ID_ADMINISTRATOR)) {
+    return;
+  }
+
+  // Remove request message and invitation option when adding a user to a group.
+  $form['membership_fields'] = FALSE;
+
+  // Add a checkbox for sending a welcome message.
+  $form['og_user']['notifications'] = array(
+    "#type" => "checkboxes",
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/c4m_user_notifications.module]: @ybaras 
please use `'#type' => 'checkbox',`

also, replace all double quotes with single quotes.

**Pull request merged**